src/Features/Core/Portable/ExternalAccess/UnitTesting/API/IUnitTestingSolutionCrawlerRegistrationServiceAccessor.cs

```diff
@@ -2,12 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.CodeAnalysis.Host;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
-    internal interface IUnitTestingSolutionCrawlerRegistrationServiceAccessor : IWorkspaceService
+    internal interface IUnitTestingSolutionCrawlerServiceAccessor : IWorkspaceService
```

Comment:
It might be good to have some comments to indicate it is assumed that the AddAnalyzerProvider() can only be called once